modules/jaggery-apps/portal/conf/site.json

```diff
@@ -1,8 +1,8 @@
 {
     "proxy" : {
-        "proxyHost" : "",
-        "proxyHTTPSPort" : "",
-        "proxyContextPath" : ""
+        "serverOrigin" : "https://localhost:9443",
+        "proxyContextPath" : "",
+        "servicePath" : "/services"
```

Comment:
IMO, having serverOrigin is enough. For me it seems more intuitive if we rename it as 'serviceOrigin'
Other configs were not used anyway as at now and servicesPath may differ when it comes to REST APIs.